src/auth/strategies/jwt.strategy.ts

```diff
@@ -18,7 +21,8 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     })
   }
 
-  async validate({ sub, username }): Promise<UserWithoutPassword> {
-    return { _id: sub, username: username }
+  validate(payload: UserJwtPayload): UserWithoutPassword {
+    delete payload.sub
```

Comment:
Let's avoid mutations. It's better cloning object first, for example:
```TS
  const clone = globalThis.structuredClone(payload);
  delete clone.sub;
  return clone;
```

Reply:
Agreed, fixed!

`structuredClone` used from the `globalThis` object instead of simple `structuredClone` usage to avoid situations when some possible local functions with the same name are also defined in this scope?